auth_user/data/res_user.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<record id="auth_user" model="res.users">
```

Comment:
`force_create=True`

Reply:
What's this?

Reply:
If data is accidentally deleted, next module update will recreate the user. Since you don't account for possibility of this user not being there it needs a force create